daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md

```diff
@@ -0,0 +1,55 @@
+---
+type: docs
+title: "Actors activation performance"
+linkTitle: "Actors activation performance"
+weight: 10000
+description: ""
+---
+This article provides service invocation API performance benchmarks and resource utilization for Actor in Dapr on Kubernetes. 
```

Comment:
"utilization for an Actor" or "utilization for Actors"

Reply:
Actors - plural